toss_3_x86_64_ib/compilers.yaml

```diff
@@ -240,10 +240,7 @@ compilers::
       cxx: /usr/tce/packages/intel/intel-16.0.4/bin/icpc
       f77: /usr/tce/packages/intel/intel-16.0.4/bin/ifort
       fc: /usr/tce/packages/intel/intel-16.0.4/bin/ifort
-    flags:
-      cflags: -gcc-name=/usr/tce/packages/gcc/gcc-7.3.0/bin/gcc
-      cxxflags: -gcc-name=/usr/tce/packages/gcc/gcc-7.3.0/bin/g++
-      fflags: -gcc-name=/usr/tce/packages/gcc/gcc-7.3.0/bin/gcc
+    flags: {}
```

Comment:
@rhornung67 @davidbeckingsale 
Just to confirm: 
What I liked about setting those here was that users can immediately see what gcc version we pair to the compiler in the RADIUSS CI, without exploring the CI specs.
I agree that the new "==" syntax, which propagates the flags, removes the need for the above "tweaking", but I want to make sure we agree on the fact that doing so we don’t specify the full vetted toolchain in radiuss-spack-configs.

Reply:
So I removed those after a brief conversation with @davidbeckingsale. I can add them back. 

I understand you concern. However, we will have an inconsistency b/c we will add shared CI specs to test clang with and without a gcc toolchain because important users of these projects build their applications that way. Can specs defined this way here be overridden by individual projects to change to a different toolchain version?

Reply:
@adrienbernede  regarding the date suffix for the older clang-ibm compilers: I don't think we need to change what's here because nobody uses those old compilers and the newer ones are named on the system by clang version rather than date. If we decide to cull out old specs, those will no longer be here.

Reply:
@adrienbernede I reverted the commit where I removed the gcc toolchain stuff.

Reply:
@rhornung67 if you need the the a compiler both with and without gcc toolchain, I suggest you do something like what spack suggests in its doc: use a version suffix.
You’d have the no-toolchain compiler with spec `intel@16.0.4`, and the one with toolchain `intel@16.0.4.7.3.0` or `intel@16.0.4.gcc.7.3.0`.

Reply:
That works too. Please feel free to rename the specs with the toolchain specified.